packages/next-templates/src/app/meldingen/graffiti-melding/form/page.tsx

```diff
@@ -28,15 +29,38 @@ import BacklinkIcon from '@/app/styling/assets/backlink-icon.svg';
 import { useForm } from 'react-hook-form';
 import { ExampleHeaderFunnel } from '@/components/ExampleHeader/ExampleHeaderFunnel/ExampleHeaderFunnel';
 import { ExampleFooterFocus } from '@/components/ExampleFooter/ExampleFooterFocus/ExampleFooterFocus';
+import { FormFieldTextarea } from '@/components/FormFieldTextarea';
+import { FormFieldTextbox } from '@/components/FormFieldTextbox';
+import {
+  messageValidation,
+  homeTownValidation,
+  streetValidation,
+  nameValidation,
+  emailValidation,
+  phoneValidation,
+} from '@/utils/validation';
 
 export default function Home() {
   const {
     register,
     handleSubmit,
     watch,
     formState: { errors },
-  } = useForm<{ [key: string]: string }>();
-  console.log(watch());
+  } = useForm<{ [key: string]: string }>({ reValidateMode: 'onBlur' });
+
+  const descriptionField = register('description', {
+    required: 'Vul een omschrijving in',
+  });
+  const placeField = register('place', homeTownValidation);
+  const streetField = register('street', streetValidation);
+  const heightField = register('height', {
+    required: 'Vul een hoogte in',
+    minLength: { value: 4, message: 'Min length is 4' },
```

Comment:
waarom is de minimum lengte 4 een huis nummer kan ook 1 getal zijn en je fout melding is in het engels niet Nederlands. de site staat in het Nederlands